examples/sklearn_elasticnet_wine/train.py

```diff
@@ -72,4 +72,15 @@ def eval_metrics(actual, pred):
         mlflow.log_metric("r2", r2)
         mlflow.log_metric("mae", mae)
 
-        mlflow.sklearn.log_model(lr, "model")
+        tracking_url_type_store = mlflow.get_tracking_uri().split(":")[0]
```

Comment:
Cool, thanks! another way to do this would be 
```
urllib.parse.urlparse(mlflow.get_tracking_uri()).scheme
```
but this look okay to me in an example context.

Reply:
That's much more cleaner.
I prefer to change that to your suggestion if you don't mind :)
Thanks.